File: Exercicios-Beecrowd/Python/1048.py
x=float(input())
if x <=400:
    print(f'Novo salario: {(x*0.15)+x:.2f}')
    print(f'Reajuste ganho: {x*0.15:.2f}')
    print(f'Em percentual: 15 %')

elif x>400 and x<=800:
    print(f'Novo salario: {(x*0.12)+x:.2f}')
    print(f'Reajuste ganho: {x*0.12:.2f}')
    print(f'Em percentual: 12 %')

elif x>800 and x<=1200:
    print(f'Novo salario: {(x*0.10)+x:.2f}')
    print(f'Reajuste ganho: {x*0.10:.2f}')
    print(f'Em percentual: 10 %')
elif x>1200 and x<=2000:
    print(f'Novo salario: {(x*0.07)+x:.2f}')
    print(f'Reajuste ganho: {x*0.07:.2f}')
    print(f'Em percentual: 7 %')
else:
    print(f'Novo salario: {(x*0.04)+x:.2f}')
    print(f'Reajuste ganho: {x*0.04:.2f}')
    print(f'Em percentual: 4 %')

File: Exercicios-Beecrowd/Kotlin/1048.kt
import java.util.*

fun main(args: Array<String>) {

    val salario= readLine().toString().toDouble()

    var porcentagem=0.0
    val dicionario= mapOf(400.00 to 0.15,400.01 to 0.12,800.01 to 0.10,1200.01 to 0.07,2000.01 to 0.04)
    for ((x,v) in dicionario){
        if (x<=salario){
            porcentagem=v

        }
        else{
            break
        }

    }
    println("Novo salario: ${"%.2f".format(((salario*porcentagem)+salario)).replace(",",".")}")
    println("Reajuste ganho: ${"%.2f".format(salario*porcentagem).replace(",",".")}")
    println("Em percentual: ${(porcentagem*100).toInt()} %")
    
	
}